src/Microsoft.ML.SamplesUtils/ConsoleUtils.cs

```diff
@@ -47,5 +48,25 @@ public static void PrintMetrics(RegressionMetrics metrics)
             Console.WriteLine($"RMS: {metrics.Rms:F2}");
             Console.WriteLine($"RSquared: {metrics.RSquared:F2}");
         }
+
+        /// <summary>
+        /// Pretty-print RankerMetrics objects.
+        /// </summary>
+        /// <param name="metrics">Ranker metrics.</param>
+        public static void PrintMetrics(RankingMetrics metrics)
+        {
+            Console.WriteLine($"DCG: {string.Join(", ", RoundAndBeautifyRankerMetrics(metrics.Dcg))}");
+            Console.WriteLine($"NDCG: {string.Join(", ", RoundAndBeautifyRankerMetrics(metrics.Ndcg))}");
+        }
+
+        private static string[] RoundAndBeautifyRankerMetrics(double[] input)
+        {
+            string[] result = input.Select(d => Math.Round(d, 2).ToString()).ToArray();
+            for (int i = 0; i < result.Length; i++)
+            {
+                result[i] = $"@{(i + 1).ToString()}:{result[i]}";
+            }
+            return result;
```

Comment:
i think you can replace this with just one line:
return input.Select((d, i) => $"@{i + 1}:{d:F2}").ToArray();